components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -417,9 +417,9 @@ lnd/clm2/surfdata_map/surfdata_ne120np4_simyr1850_c160211.nc</fsurdat>
 <fsurdat hgrid="ne11np4"      sim_year="1850" use_crop=".false." >
 lnd/clm2/surfdata_map/surfdata_ne11np4_simyr1850_c160614_with_TOP.nc</fsurdat>
 <fsurdat hgrid="ne4np4"      sim_year="1850" use_crop=".false." >
-lnd/clm2/surfdata_map/surfdata_ne4np4_simyr1850_c160614.nc</fsurdat>
-<fsurdat hgrid="ne4np4.pg2"  sim_year="1850" >
-lnd/clm2/surfdata_map/surfdata_ne4pg2_simyr1850_c210722.nc</fsurdat>
+lnd/clm2/surfdata_map/surfdata_ne4np4_simyr1850_c160614_with_TOP.nc</fsurdat>
```

Comment:
Instead of "with_TOP" change the date string at the end do 20231019 or other nearby date.

Reply:
For easy recognition, all this series of surfdata files are simply with addition of TOP related variables to the original file. The current addition are placeholders in order for default config (v3alpha04 and later) to work. When they are eventually updated by the TOP developer, the final cdates will be attached.

Reply:
ok